### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by silbinarywolf** [avocode/avocode#1760]
---
* **Version**: 2.17.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 203963
---
```
TypeError: Cannot read property 'toJS' of undefined
    at a (file:///C:/Users/jake/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:1379796)
    at Object.s [as convertColor] (file:///C:/Users/jake/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:1379937)
    at file:///C:/Users/jake/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:112562
    at C:\Users\jake\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:3018:46
    at List.__iterate (C:\Users\jake\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:2208:13)
    at IndexedIterable.mappedSequence.__iterateUncached (C:\Users\jake\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:3017:23)
    at seqIterate (C:\Users\jake\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:606:16)
    at IndexedI
```

**Comment by jakubzitny** [comment]
Hi @silbinarywolf, thanks for reporting this. This is a bug, I can see it in our tracking system. We'll fi it as soon as possbile.

**Comment by jakubzitny** [comment]
Hi guys, this should be already fixed in `2.17.3`, please give it a try and let us know if you have any more problems.
